tests/test_datasource_version.py

```diff
@@ -0,0 +1,60 @@
+import json
+import mock
+
+from tests import BaseTestCase
+from flask import Flask
+
+from redash.models import DataSource
+from redash.query_runner.pg import PostgreSQL
+from redash_stmo.datasource_version import datasource_version
+
+
+class TestDatasourceVersion(BaseTestCase):
+    EXPECTED_DOC_URL = "www.example.com"
+    def setUp(self):
+        super(TestDatasourceVersion, self).setUp()
+        self.admin = self.factory.create_admin()
+        self.data_source = self.factory.create_data_source()
+        self.patched_run_query = self._setup_mock('redash.query_runner.pg.PostgreSQL.run_query')
+        datasource_version(self.app)
+
+    def _setup_mock(self, function_to_patch):
+        patcher = mock.patch(function_to_patch)
+        patched_function = patcher.start()
+        self.addCleanup(patcher.stop)
+        return patched_function
+
+    def _test_expected_version_returned(self, expected_version, version_string, runner_type):
```

Comment:
Is this method picked up by the test runner with that underscore in the name?

Reply:
Nope. it's a helper method called by the other test methods and the runner doesn't pick it up.

Reply:
D'oh!